mkosi/backend.py

```diff
@@ -627,6 +627,10 @@ def nspawn_rlimit_params() -> Sequence[str]:
     ]
 
 
+def nspawn_executable() -> Path:
```

Comment:
This is a silly nit, feel free to ignore it, but the fallback is not really a path, but a string that will be looked up in `PATH` to get a path. I think `str` as a return type would express this better.